include/lmms_math.h

```diff
@@ -96,6 +96,57 @@ static void roundAt(T& value, const T& where, const T& stepSize)
 	}
 }
 
+//! Uses `x * y + z` or `std::fma` - whichever is faster
```

Comment:
I've got a different idea for `fastFma` functions. Instead of doing this, how about using `std::fma` directly in `interpolation.h`, but explicitly define `FP_FAST_FMA*` with define guards? 
This idea came to me after the discussion on discord.

Reply:
Oops mb. It's fine then.